File: app/src/main/java/com/example/schooldiary/ui/bottomNavigation/ShowNewsFragment.java
package com.example.schooldiary.ui.bottomNavigation;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.bumptech.glide.Glide;
import com.example.schooldiary.R;
import com.example.schooldiary.databinding.FragmentShowNewsBinding;


public class ShowNewsFragment extends Fragment {

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_show_news, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {

        FragmentShowNewsBinding binding = FragmentShowNewsBinding.bind(view);
        ImageView newsPhoto = binding.newsPhoto;
        TextView newsTime = binding.newsTime;
        TextView newsTitle = binding.newsTitle;
        TextView newsText = binding.newsText;

        Bundle bundle = this.getArguments();
        if (bundle != null) {

            String photo = bundle.getString("Photo");
            Glide.with(newsPhoto.getContext())
                    .load(photo)
                    .into(newsPhoto);

            String time = bundle.getString("Time");
            newsTime.setText(time);

            String title = bundle.getString("Title");
            newsTitle.setText(title);

            String text = bundle.getString("Text");
            newsText.setText(text);

        }
    }
}

File: app/src/main/java/com/example/schooldiary/ui/bottomNavigation/ShowNewsFragment.kt
package com.example.schooldiary.ui.bottomNavigation

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.bumptech.glide.Glide
import com.example.schooldiary.databinding.FragmentShowNewsBinding
import com.example.schooldiary.model.News

class ShowNewsFragment : Fragment() {
    private var _binding: FragmentShowNewsBinding? = null
    private val binding get() = _binding!!

    private var news: News? = null

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        _binding = FragmentShowNewsBinding.inflate(inflater, container, false)
        return binding.root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        arguments?.let {
            news = it.getParcelable(ARGS_NEWS)
        }

        news?.let {
            if (it.photo != UNKNOWN) {
                Glide.with(binding.photo.context)
                        .load(it.photo)
                        .into(binding.photo)
            }
            binding.time.text = it.time
            binding.title.text = it.title
            binding.text.text = it.text
        }
    }

    override fun onDestroyView() {
        super.onDestroyView()
        _binding = null
    }

    companion object {
        private const val UNKNOWN = "Unknown"
        private const val ARGS_NEWS = "news"
    }
}